### ISSUE TITLE
Export for DirectDrawEnumerateExA is missing.

### ISSUE THREAD

**Issue by zhro** [jarikomppa/dxwrapper#1]
There is no export for DirectDrawEnumerateExA. Specifically, I am trying to run a DirectX 7 application.

**Comment by jarikomppa** [comment]
You're right. There's apparently only DirectDrawEnumerateA. The other enumerate functions probably follow similar pattern, though, so you may want to do a copy of myDirectDrawEnumerate in ddinput.cpp and modify that to the Ex version.
